requirements.txt

```diff
@@ -1,2 +1,2 @@
 sqlalchemy>=1.1.0
-hypothesis>=4.22.0
+hypothesis>=4.53.2
```

Comment:
I've tested those changes and they seem to work on `4.22.0` version as well, can we leave it as it was?

Reply:
Sure :)